R/timeseries_plot.R

```diff
@@ -0,0 +1,98 @@
+#' Title
+#'
+#' @param data 
+#' @param time 
+#' @param elements 
+#' @param station 
+#' @param facets 
+#' @param add_points 
+#' @param add_line_of_best_fit 
+#' @param se 
+#' @param add_path 
+#' @param add_step 
+#' @param na.rm 
+#' @param show.legend 
+#'
+#' @return
+#' @export
+#'
+#' @examples
+timeseries_plot <- function(data, time, elements, station = NULL, facets = c("none", "elements", "stations", "both"),
```

Comment:
I think `stations` should be the default for `facets`

Reply:
Yeah good point, let's say if station isn't provided give a message saying setting facets to none.